### PR TITLE
jupyter: update to version 201214

### PR DESCRIPTION
**Pull request by tlvu**
Matching PR to deploy the new Jupyter env in PR https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/56 to PAVICS.

Relevant changes:

```diff
>   - cfgrib=0.9.8.5=pyhd8ed1ab_0

<   - clisops=0.3.1=pyh32f6830_1
>   - clisops=0.4.0=pyhd3deb0d_0

<   - dask=2.30.0=py_0
>   - dask=2020.12.0=pyhd8ed1ab_0

<   - owslib=0.20.0=py_0
>   - owslib=0.21.0=pyhd8ed1ab_0

<   - xarray=0.16.1=py_0
>   - xarray=0.16.2=pyhd8ed1ab_0

<   - xclim=0.21.0=py_0
>   - xclim=0.22.0=pyhd8ed1ab_0

<   - jupyter_conda=3.4.1=pyh9f0ad1d_0
>   - jupyter_conda=4.1.0=hd8ed1ab_1
```

### CODE REVIEW

**Comment by tlvu** [comment]
@tlogan2000 sorry, can you approve for this repo.  I can not merge without approval.  It's deployed on Medus if you want to try out.

**Review by tlogan2000 — APPROVED** [review]

**Pull request merged**
